### ISSUE TITLE
Update issue

### ISSUE THREAD

**Issue by psaunders** [felixkiss/uniquewith-validator#33]
The validator is firing for me with any update because I have Model level validation. Could you add something to detect that the validation should not run if the id of the model is the duplicate it is detecting.

**Comment by SaifurRahmanMohsin** [comment]
+1 same issue. Is there a solution for this?

**Comment by felixkiss** [comment]
How is your validation called? Do you use a model validation package or roll your own code?

**Comment by SaifurRahmanMohsin** [comment]
I’m using model validation

```
    public $rules = [
        'user' => 'required',
        'establishment'  => 'required',
        'user_id' => 'unique_with:mohsin_rating_reviews, establishment_id'
    ];
```

In my app, this was added to the **Review** model and the logic here is that an **establishment** can have only one review from a particular **user**. Both **user** and **establishment** are `belongsTo` Eloquent relationships.

**Comment by felixkiss** [comment]
Right. But what package are you using to get the model validation functionality? This is important to know how to tackle your problem.

**Comment by SaifurRahmanMohsin** [comment]
I’m not entirely sure what specifically you are asking for. I use the one that comes with [Laravel 5](http://laravel.com/docs/5.0/validation#available-validation-rules)

**Comment by felixkiss** [comment]
I am not aware of a model validation feature provided by Laravel out-of-the-box. How does your Model look? Does it extend `Illuminate\Database\Eloquent\Model` directly?

**Comment by SaifurRahmanMohsin** [comment]
Yes

**Comment by felixkiss** [comment]
Closing because of inactivity
